components/player/playerTypeList.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styled from 'styled-components';
+import { IPlayer, IPlayerDetails } from '../../interfaces';
+import PlayerList from './playerList';
+
+const PlayerTypeContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 50%;
+
+  @media(max-width: 1200px){
+    width: 100%;
+  }
+`;
+
+const MetaDataWrapper = styled.div`
+  display: flex;
+  margin-Bottom: 0.5rem
+`;
+
+const MetaData = styled.p`
+  font-weight: bold;
+  min-width: 10rem;
+  max-width: 30rem;
+`;
+
+interface IPlayerTypeList {
+  playerDetails: IPlayerDetails,
+  selectPlayer: (guid: number) => void;
+}
+
+const PlayerTypeList = ({ playerDetails, selectPlayer }: IPlayerTypeList) => {
+  const metaData = ['Name', 'Class', 'ILvl', 'Spec'];
+  return (
+    <PlayerTypeContainer>
+      {Object.keys(playerDetails)
+        .reverse()
```

Comment:
this definitely needs documentation, its not clear what that actually achieves in the context of the component

Reply:
todo